client/my-sites/stats/stats-subscribers-page/subsriber-highlights.jsx

```diff
@@ -0,0 +1,97 @@
+import { HighlightCard, Popover } from '@automattic/components';
```

Comment:
This file name has a typo; it should probably be `subscriber-highlights.jsx`.

Reply:
I actually thought about naming the component `SubscribersHighlights` but felt it read awkwardly. I think you're right though, as it makes for a more consistent naming pattern within the page. I'll address the typo as well.